is_core/utils/__init__.py

```diff
@@ -271,17 +272,36 @@ def display_for_value(value, request=None):
         boolean True/Talse ==> Yes/No
         objects ==> object display name with link if current user has permissions to see the object
         datetime ==> in localized format
+        list ==> values separated with ","
+        dict ==> string formatted with HTML ul/li tags
     """
 
     if request and isinstance(value, Model):
         return render_model_object_with_link(request, value)
-    if request and isinstance(value, QuerySet):
-        return render_model_objects_with_link(request, value)
-    else:
-        return (
-            (value and ugettext('Yes') or ugettext('No')) if isinstance(value, bool)
-            else admin_display_for_value(value, '-')
+    elif isinstance(value, (QuerySet, list, tuple, set, types.GeneratorType)):
+        return format_html_join(', ', '{}', ((display_for_value(v, request),) for v in value))
+    elif isinstance(value, dict):
+        return format_html(
+            '<ul class="field-dict">{}</ul>',
+            format_html_join(
+                '\n',
+                '{}{}',
+                (
+                    (
+                        format_html('<li>{}</li>', k),
+                        (
+                            display_for_value(v, request) if isinstance(v, dict)
```

Comment:
Zdá se mi, že máš prohozené if - else.

Reply:
nemám. Je to správně. Když je uvnitř dict tak chci to nechci obalovat ul ani li, protože pak bude v ul,li zas ul a to je špatně